macro-kroki-ui/src/main/resources/XWiki/Kroki/ConfigTemplate.xml

```diff
@@ -251,6 +266,9 @@
     <property>
       <host/>
     </property>
+    <property>
+      <httpProtocol/>
```

Comment:
Where doesn't this come from? It should have been ``useTLS`` no?

Reply:
Yes, I initially used a different name before deciding that useTLS would be more fitting, so I must have missed this one when changing my implementation.